app/assets/stylesheets/settings/_settings.scss

```diff
@@ -40,6 +40,5 @@ $color-6: $dark-green-blue;
 
 
 /* FONT COLORS */
-$font-color-1: $black;
-$font-color-2: $white;
-
+$font-color-1: $white;
+$font-color-2: $black;
```

Comment:
Agree to add here the colors for the text, but please, in this case, replace them in the text file.